core/javaAction/Dockerfile

```diff
@@ -1,38 +1,11 @@
-FROM buildpack-deps:trusty-curl
-
-ENV DEBIAN_FRONTEND noninteractive
-
-RUN locale-gen en_US.UTF-8
-ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US:en
-ENV LC_ALL en_US.UTF-8
-
```

Comment:
You’ll want to restore these.

Reply:
I haven’t looked at the new dockerfile dependence below. If these aren’t set you’ll need to reset them.

Reply:
Yep those are the ones I was referring 👍

Reply:
Restored the locale specific environment variable settings.

Reply:
Thanks  @rabbah @csantanap Restoring the environment variable solved the issue. All checks passed.

Reply:
Just noticed you added the locales package here.  Good belt-and-suspenders approach.